engine/src/main/scala/cromwell/engine/workflow/lifecycle/finalization/WorkflowFinalizationActor.scala

```diff
@@ -99,7 +102,8 @@ case class WorkflowFinalizationActor(workflowDescriptor: EngineWorkflowDescripto
         } yield actor
       }
 
-      val engineFinalizationActor = Try { copyWorkflowOutputsActorProps.map(context.actorOf(_, "CopyWorkflowOutputsActor")).toList }
+      val engineFinalizationActor = Try {copyWorkflowOutputsActorProps.map(context.actorOf(_, "CopyWorkflowOutputsActor")).toList :::
```

Comment:
I think if you are using multiline `Try`, first line of expression should be on the next line after opening bracket, and a closing bracket also should be on a new line.

Reply:
Totally